WinUIGallery/DataModel/ControlInfoData.json

```diff
@@ -2900,14 +2900,31 @@
         },
         {
           "UniqueId": "SystemBackdrops",
-          "Title": "System Backdrops (Mica/Acrylic)",
+          "Title": "SystemBackdrop",
```

Comment:
Please revert this since including Mica and Acrylic in the name helps with this page being found in the search when looking for those terms.

Reply:
Agree, this also makes it easier to discover for UWP devs